nodego/env.go

```diff
@@ -0,0 +1,43 @@
+package nodego
+
+import (
+	"os"
+	"strconv"
+	"time"
+)
+
+var (
+	_CODE_LOCATION_DIR        = os.Getenv("CODE_LOCATION")
+	_PACKAGE_JSON_FILE        = _CODE_LOCATION_DIR + "/package.json"
+	_ENTRY_POINT              = os.Getenv("ENTRY_POINT")
+	_SUPERVISOR_HOSTNAME      = os.Getenv("SUPERVISOR_HOSTNAME")
+	_SUPERVISOR_INTERNAL_PORT = os.Getenv("SUPERVISOR_INTERNAL_PORT")
+	_FUNCTION_TRIGGER_TYPE    = os.Getenv("FUNCTION_TRIGGER_TYPE")
+	_FUNCTION_NAME            = os.Getenv("FUNCTION_NAME")
+	_FUNCTION_TIMEOUT_SEC, _  = strconv.ParseInt(os.Getenv("FUNCTION_TIMEOUT_SEC"), 10, 64)
+	// HTTP header field that is added to Worker response to signalize problems with
```

Comment:
Comments documenting declarations should begin with the name of the thing being described and end in a period.
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
Should these comments be left out and replaced with a reference to the `worker.js` file?